content/developer/reference/backend/views.rst

```diff
@@ -178,6 +178,13 @@ an impact on all view types.
 
 .. todo:: Views main content section, with field, group & separator ?
 
+* ``class``
+  optional string containing some classes to set on the root node of the view.
+
+* ``js_class``
+  optional view registry key determining the component class to use to render the view.
+  If set, the class ``o_'js_class'_view`` will be set on the root node of the view.
```

Comment:
how to write ``o_'js_class'_view`` well?
I mean that js_class = custom_kanban would produce o_custom_kanban_view for instance

Reply:
Something like this maybe ?

If set, the `js_class` value will be used to replace the placeholder in the class name `o_{js_class}_view`, and this modified class name will be applied to the root node of the view. For example, if `js_class` is set to `my-custom-kanban`, the class `o_my-custom-kanban_view` will be used.

Reply:
thank you